src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -240,5 +239,15 @@ private static string GetExpectedRedirectUri(string clientId)
         {
             return $"ms-appx-web://microsoft.aad.brokerplugin/{clientId}";
         }
+
+        /// <summary>
+        /// Get ExpiresIn from Auth Result
+        /// </summary>
+        /// <param name="expiresOn"></param>
+        private static long GetExpiresInFromAuthResult(DateTime expiresOn)
```

Comment:
Is it possible to get this covered by an integration test? Maybe it's worth waiting a bit until we have an integration test.

Reply:
merging to get this in for this week's release. Will add a test when the Integration pipeline is ready